src/kontrol/prove.py

```diff
@@ -852,13 +896,17 @@ def _create_initial_account_list(
 def _final_cterm(
     empty_config: KInner,
     program: KInner,
+    config_type: ConfigType,
     *,
     failing: bool,
     is_test: bool = True,
-    is_setup: bool = False,
     hevm: bool = False,
 ) -> CTerm:
-    final_term = _final_term(empty_config, program, is_test=is_test, is_setup=is_setup)
+    final_term = _final_term(
+        empty_config,
+        program,
+        config_type=config_type,
+    )
```

Comment:
This is perhaps a bit silly, but why does this get broken down over four lines when the previous version didn't and this one has fewer characters?

Reply:
I'm not sure. `black` seems to be happy with either way, so I've manually changed it to the one line version.